lib/rspec/rails/matchers/have_enqueued_mail.rb

```diff
@@ -1,4 +1,4 @@
-require "rspec/mocks"
+require "rspec/mocks/argument_matchers"
```

Comment:
```suggestion
# We require the minimum amount of rspec-mocks possible to avoid conflicts with over mocking frameworks.
# See: https://github.com/rspec/rspec-rails/issues/2252
require "rspec/mocks/argument_matchers"
```